js/src/forum/generated/emojiMap.json

```diff
@@ -0,0 +1 @@
"🇬🇱":["flag_gl"],"🇬🇲":["flag_gm"],"🇬🇳":["flag_gn"],"🇬🇵":["flag_gp"],"🇬🇶":["flag_gq"],"🇬🇷":["flag_gr"],"🇬🇸":["flag_gs"],"🇬🇹":["flag_gt"],"🇬🇺":["flag_gu"],"🇬🇼":["flag_gw"],"🇬🇾":["flag_gy"],"🇭🇰":["flag_hk"],"🇭🇲":["flag_hm"],"🇭🇳":["flag_hn"],"🇭🇷":["flag_hr"],"🇭🇹":["flag_ht"],"🇭🇺":["flag_hu"],"🇮🇨":["flag_ic"],"🇮🇩":["flag_id"],"🇮🇪":["flag_ie"],"🇮🇱":["flag_il"],"🇮🇲":["flag_im"],"🇮🇳":["flag_in"],"🇮🇴":["flag_io"],"🇮🇶":["flag_iq"],"🇮🇷":["flag_ir"],"🇮🇸":["flag_is"],"🇮🇹":["flag_it"],"🇯🇪":["flag_je"],"🇯🇲":["flag_jm"],"🇯🇴":["flag_jo"],"🇯🇵":["flag_jp"],"🇰🇪":["flag_ke"],"🇰🇬":["flag_kg"],"🇰🇭":["flag_kh"],"🇰🇮":["flag_ki"],"🇰🇲":["flag_km"],"🇰🇳":["flag_kn"],"🇰🇵":["flag_kp"],"🇰🇷":["flag_kr"],"🇰🇼":["flag_kw"],"🇰🇾":["flag_ky"],"🇰🇿":["flag_kz"],"🇱🇦":["flag_la"],"🇱🇧":["flag_lb"],"🇱🇨":["flag_lc"],"🇱🇮":["flag_li"],"🇱🇰":["flag_lk"],"🇱🇷":["flag_lr"],"🇱🇸":["flag_ls"],"🇱🇹":["flag_lt"],"🇱🇺":["flag_lu"],"🇱🇻":["flag_lv"],"🇱🇾":["flag_ly"],"🇲🇦":["flag_ma"],"🇲🇨":["flag_mc"],"🇲🇩":["flag_md"],"🇲🇪":["flag_me"],"🇲🇫":["flag_mf"],"🇲🇬":["flag_mg"],"🇲🇭":["flag_mh"],"🇲🇰":["flag_mk"],"🇲🇱":["flag_ml"],"🇲🇲":["flag_mm"],"🇲🇳":["flag_mn"],"🇲🇴":["flag_mo"],"🇲🇵":["flag_mp"],"🇲🇶":["flag_mq"],"🇲🇷":["flag_mr"],"🇲🇸":["flag_ms"],"🇲🇹":["flag_mt"],"🇲🇺":["flag_mu"],"🇲🇻":["flag_mv"],"🇲🇼":["flag_mw"],"🇲🇽":["flag_mx"],"🇲🇾":["flag_my"],"🇲🇿":["flag_mz"],"🇳🇦":["flag_na"],"🇳🇨":["flag_nc"],"🇳🇪":["flag_ne"],"🇳🇫":["flag_nf"],"🇳🇬":["flag_ng"],"🇳🇮":["flag_ni"],"🇳🇱":["flag_nl"],"🇳🇴":["flag_no"],"🇳🇵":["flag_np"],"🇳🇷":["flag_nr"],"🇳🇺":["flag_nu"],"🇳🇿":["flag_nz"],"🇴🇲":["flag_om"],"🇵🇦":["flag_pa"],"🇵🇪":["flag_pe"],"🇵🇫":["flag_pf"],"🇵🇬":["flag_pg"],"🇵🇭":["flag_ph"],"🇵🇰":["flag_pk"],"🇵🇱":["flag_pl"],"🇵🇲":["flag_pm"],"🇵🇳":["flag_pn"],"🇵🇷":["flag_pr"],"🇵🇸":["flag_ps"],"🇵🇹":["flag_pt"],"🇵🇼":["flag_pw"],"🇵🇾":["flag_py"],"🇶🇦":["flag_qa"],"🇷🇪":["flag_re"],"🇷🇴":["flag_ro"],"🇷🇸":["flag_rs"],"🇷🇺":["flag_ru"],"🇷🇼":["flag_rw"],"🇸🇦":["flag_sa"],"🇸🇧":["flag_sb"],"🇸🇨":["flag_sc"],"🇸🇩":["flag_sd"],"🇸🇪":["flag_se"],"🇸🇬":["flag_sg"],"🇸🇭":["flag_sh"],"🇸🇮":["flag_si"],"🇸🇯":["flag_sj"],"🇸🇰":["flag_sk"],"🇸🇱":["flag_sl"],"🇸🇲":["flag_sm"],"🇸🇳":["flag_sn"],"🇸🇴":["flag_so"],"🇸🇷":["flag_sr"],"🇸🇸":["flag_ss"],"🇸🇹":["flag_st"],"🇸🇻":["flag_sv"],"🇸🇽":["flag_sx"],"🇸🇾":["flag_sy"],"🇸🇿":["flag_sz"],"🇹🇦":["flag_ta"],"🇹🇨":["flag_tc"],"🇹🇩":["flag_td"],"🇹🇫":["flag_tf"],"🇹🇬":["flag_tg"],"🇹🇭":["flag_th"],"🇹🇯":["flag_tj"],"🇹🇰":["flag_tk"],"🇹🇱":["flag_tl"],"🇹🇲":["flag_tm"],"🇹🇳":["flag_tn"],"🇹🇴":["flag_to"],"🇹🇷":["flag_tr"],"🇹🇹":["flag_tt"],"🇹🇻":["flag_tv"],"🇹🇼":["flag_tw"],"🇹🇿":["flag_tz"],"🇺🇦":["flag_ua"],"🇺🇬":["flag_ug"],"🇺🇲":["flag_um"],"🇺🇳":["flag_un"],"🇺🇸":["flag_us","usa"],"🇺🇾":["flag_uy"],"🇺🇿":["flag_uz"],"🇻🇦":["flag_va"],"🇻🇨":["flag_vc"],"🇻🇪":["flag_ve"],"🇻🇬":["flag_vg"],"🇻🇮":["flag_vi"],"🇻🇳":["flag_vn"],"🇻🇺":["flag_vu"],"🇼🇫":["flag_wf"],"🇼🇸":["flag_ws"],"🇽🇰":["flag_xk"],"🇾🇪":["flag_ye"],"🇾🇹":["flag_yt"],"🇿🇦":["flag_za"],"🇿🇲":["flag_zm"],"🇿🇼":["flag_zw"],"🏴󠁧󠁢󠁥󠁮󠁧󠁿":["flag_gbeng","england"],"🏴󠁧󠁢󠁳󠁣󠁴󠁿":["flag_gbsct","scotland"],"🏴󠁧󠁢󠁷󠁬󠁳󠁿":["flag_gbwls","wales"]}
```

Comment:
This file probably doesn't need to be committed? Since it is generated when building

Reply:
should I add `mkdir src/forum/generated` to build command or should I add `.gitkeep`?

Reply:
or maybe script should make that folder too?